test/o-error-util.test.js

```diff
@@ -149,9 +149,44 @@ describe('OError.tag', function () {
       expect.fail('should have yielded an error')
     })
   })
+
+  it('is not included in the stack trace if using capture', function () {
+    if (!Error.captureStackTrace) return
```

Comment:
```suggestion
    if (!Error.captureStackTrace) return this.skip()
```
Let's flag that we are not evaluating this test case when `Error.captureStackTrace` is missing. [Mocha docs are here.](https://mochajs.org/#inclusive-tests)